QA_Requests/PerfTickets/testCases/test_capture_cle_perf.py

```diff
@@ -0,0 +1,13 @@
+from QA_Requests.PerfTickets.testPages.cle_perf.cle import CLEPage
+
+
+def test_case_cle(driver):
+    cle = CLEPage(driver)
+    global parent_case_type, child_case_type,  load
+    load = "10k"
```

Comment:
why are these variables marked global? There seems to be only 1 testcase/method here.

Reply:
I made some changes and didn't remove those later -- Removed now https://github.com/dimagi/dimagi-qa/pull/137/commits/8fa03009d5d9714d8a4724bb34c4ee738d0710dc